game/game/states/GameState.cpp

```diff
@@ -166,7 +188,10 @@ void GameState::onUpdate(Keyboard *keyboard)
 	}
 	if (keyboard->isKeydown(KEY_W)) { player->jump(); p1LastDirection = UP; }
 	if (keyboard->isKeydown(KEY_A)) { player->setPlayerState(PLAYER_LEFT); p1LastDirection = LEFT; }
-	if (keyboard->isKeydown(KEY_D)) { player->setPlayerState(PLAYER_RIGHT); p1LastDirection = RIGHT; }
+	if (keyboard->isKeydown(KEY_D))

```

Comment:
is niet helemaal consistent met de rest. Ik zou dit op dezelfde regel houden als de rest.